openapi/src/common/parameters.yaml

```diff
@@ -80,6 +80,14 @@ components:
       schema:
         $ref: '#/components/schemas/IamRole'
 
+    SasIpRange:
+      name: sasIpRange
+      in: query
+      description: An IP address or range of addresses to restrict a SAS token to
```

Comment:
Nit: maybe add a note in the description here about how to format a range of addresses?